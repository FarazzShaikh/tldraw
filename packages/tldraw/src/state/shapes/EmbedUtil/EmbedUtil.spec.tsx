import { Embed } from '..'

describe('Embed shape', () => {
  it('Creates a shape', () => {
    expect(Embed.create({ id: 'embed' })).toMatchSnapshot('embed')
  })
})
