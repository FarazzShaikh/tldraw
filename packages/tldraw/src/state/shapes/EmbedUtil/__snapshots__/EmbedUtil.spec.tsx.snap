// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Embed shape Creates a shape: embed 1`] = `
Object {
  "assetId": "assetId",
  "childIndex": 1,
  "id": "embed",
  "name": "Embed",
  "parentId": "page",
  "point": Array [
    0,
    0,
  ],
  "rotation": 0,
  "size": Array [
    1,
    1,
  ],
  "style": Object {
    "color": "black",
    "dash": "draw",
    "isFilled": false,
    "scale": 1,
    "size": "small",
  },
  "type": "embed",
}
`;
